vsintegration/src/FSharp.Editor/Classification/ClassificationDefinitions.fs

```diff
@@ -41,6 +46,63 @@ module internal FSharpClassificationTypes =
         | SemanticClassificationType.Interface -> Interface
 
 module internal ClassificationDefinitions =
+
+    [<Export>]
+    type internal ThemeColors
+        [<ImportingConstructor>]
+        (
+            classificationformatMapService: IClassificationFormatMapService,
+            classificationTypeRegistry: IClassificationTypeRegistryService,
+            [<Import(typeof<SVsServiceProvider>)>] serviceProvider: IServiceProvider
+        ) =
+
+        let (| LighTheme | DarkTheme | UnknownTheme |) id =
+            if id = Guid("de3dbbcd-f642-433c-8353-8f1df4370aba") ||
+               id = Guid("a4d6a176-b948-4b29-8c66-53c97a1ed7d0") then LighTheme 
+            elif id = Guid("1ded0138-47ce-435e-84ef-9ec1f439b749") then DarkTheme
+            else UnknownTheme
+    
+        let getCurrentThemeId() =
+            let themeService = serviceProvider.GetService(typeof<SVsColorThemeService>) :?> IVsColorThemeService
+            themeService.CurrentTheme.ThemeId
+
+        let colorData = // name,                  (light,                            dark)
+          [ FSharpClassificationTypes.Function,   (Colors.Black,                     Color.FromRgb(220uy, 220uy, 220uy))
+            FSharpClassificationTypes.MutableVar, (Color.FromRgb(160uy, 128uy, 0uy), Color.FromRgb(255uy, 210uy, 28uy))
+            FSharpClassificationTypes.Printf,     (Color.FromRgb(43uy, 145uy, 175uy), Color.FromRgb(78uy, 220uy, 176uy))
+            FSharpClassificationTypes.Property,   (Colors.Black,                     Color.FromRgb(220uy, 220uy, 220uy)) ]
+
+        let setColors _ =
+            let fontAndColorStorage = serviceProvider.GetService(typeof<SVsFontAndColorStorage>) :?> IVsFontAndColorStorage
+            let fontAndColorCacheManager = serviceProvider.GetService(typeof<SVsFontAndColorCacheManager>) :?> IVsFontAndColorCacheManager
+            let textEditor = Guid("A27B4E24-A735-4D1D-B8E7-9716E1E3D8E0")
```

Comment:
guidTextEditorFontCategory?   Language service also uses it, perhaps a named identifier somewhere?